src/Aspire.Hosting.Azure/Bicep/cosmosdb.bicep

```
@@ -10,32 +11,43 @@ param location string = resourceGroup().location
 var resourceToken = uniqueString(resourceGroup().id)
 
 resource cosmosDb 'Microsoft.DocumentDB/databaseAccounts@2023-04-15' = {
-  name: replace('${databaseAccountName}-${resourceToken}', '-', '')
-  location: location
-  kind: 'GlobalDocumentDB'
-  tags: tags
-  properties: {
-    consistencyPolicy: { defaultConsistencyLevel: 'Session' }
-    locations: [
-      {
-        locationName: location
-        failoverPriority: 0
-      }
-    ]
-    databaseAccountOfferType: 'Standard'
-  }
-
-  resource db 'sqlDatabases@2023-04-15' = [for name in databases:{
-    name: '${name}'
+    name: replace('${databaseAccountName}-${resourceToken}', '-', '')
```

```suggestion
    name: replace('${databaseAccountName}${resourceToken}', '-', '')
```

Is it weird to add a `-` and then just remove it in the same line?

---

It is, copied from azd didn’t spend too much time cleaning up the bicep